docs/client_tools/plugins/deadline.md

```diff
@@ -0,0 +1,124 @@
+> - `/opt/autodesk/maya-io/2019/maya-io2019.SP2/bin/Render`
+
+5. Add the `submit_to_conductor.py` as a Job menu script. The path to the icon is `deadline_plugin/custom/scripts/General/conductor_logo.ico`
+![](../../image/deadline/script_menu_repo.jpg)
+
+## Usage
+### Prerequisites
+1. The Conductor upload daemon must be running
+2. The Conductor download daemon must be running
+3. A dependency sidecar exists for the given job
+
+###Sending Jobs to Conductor
+1. From the Deadline Monitor, select one or multiple jobs.
+2. From the context menu, select `Scripts->Send to Conductor`
+3. A dialog will appear where you can modify the default options for a job
+4. Once you click `OK` the job will be submitted to Conductor
+5. The dialog will reappear for each job selected (ex: if you selected five jobs to submit, the dialog will appear five times)
+6. It's imperative that a valid dependency sidecar file be selected. For more details see the section on Dependencies
+7. After a few minutes new workers pop-up in the worker panel that have the suffix `-conductor`. Each worker will render exactly one Deadline task. After a worker completes a Deadline task, they will shut down.
+8. It's safe to `Delete` the Conductor workers that have completed their task. They will have a state of `Stalled`. This can be done from the Workers panel.
+
+## How it works
+The above is achieved by running a special Deadline worker job on Conductor - which is the Deadline worker software with some parameters to ensure it shuts down after it completes excactly one Deadline task. There's some additional logic to create a temporary Deadline group (`conductorautogroup_<jobid>`) to ensure that the submitted Deadline job only runs on the Conductor instances and that the Conductor instances only pick-up the given Deadline job.
+
+An essential step is to ensure that a valid sidecar dependency file exists. If it doesn't, the Conductor instances will not have any of the necessary files to run the task. See the section on dependencies for more details.
+
+Viewing reports and most other functions of a Deadline worker will work with a Conductor deadline worker, except for any operation that involves sending a command directly to the host instance (see below for more details).
+
+<div style="width: 640px; height: 480px; margin: 10px; position: relative;"><iframe allowfullscreen frameborder="0" style="width:640px; height:480px" src="https://www.lucidchart.com/documents/embeddedchart/db7ae6b1-0106-4bef-aaa9-ff87d3558dda" id=".01_Fkrc4X2p"></iframe></div>
+
+
+## Dependencies
+A task can not render if it does not have all of its required dependencies. The list of necessary dependencies are submitted to Conductor as part of the job submission process. The uploader daemon is responsible for performing the actual upload*. Therefore, it's necessary to create this list of dependencies before the job is submitted. There are several ways to do this:
```

Comment:
NIT from Grammarly (because there's one list)
`s/dependencies are submitted/dependencies is submitted/`